cores/esp32/esp32-hal-periman.c

```diff
@@ -78,7 +82,7 @@ peripheral_bus_type_t perimanGetPinBusType(uint8_t pin){
 }
 
 bool perimanSetBusDeinit(peripheral_bus_type_t type, peripheral_bus_deinit_cb_t cb){
-	if(type >= ESP32_BUS_TYPE_MAX){
+	if(type >= ESP32_BUS_TYPE_MAX || type == ESP32_BUS_TYPE_INIT){
```

Comment:
Same here as previous comment from @me-no-dev, we will never call perimanSetBusDeinit with INIT type. This check is not that necessary :) but of course is not breaking anything and can be there.

Reply:
This makes the code clear about it. As commented below.

> The added testing for `type == ESP32_BUS_TYPE_INIT` is inteded to make it clear in the code. We shall think that today we are working on it and know that we won't use it with `ESP32_BUS_TYPE_INIT`, but in the future, other people may be working on this code.
> 
> I think that all that makes the code clearer for others, including commentaries, shall be used.